Examples/create-app.js

```diff
@@ -89,7 +98,9 @@ function generatePlainReactNativeApp(appName, reactNativeVersion) {
 
 function installCodePush(reactNativeCodePushVersion) {
     console.log(`Installing React Native Module for CodePush...`);
-    execSync(`npm i --save ${reactNativeCodePushVersion}`);
+    // execSync(`npm i --save ${reactNativeCodePushVersion}`);
+    // Fixed some problems with dependencies
+    execSync(`yarn add ${reactNativeCodePushVersion}`);
```

Comment:
Don't get this, could you please clarify what problems w/ dependencies are when using npm? It seems like I usually use npm for new react-native+cp project and don't see any issues.

Reply:
When I use ```npm```, I'm getting this (when I try to build the created app on android):
```
FAILURE: Build failed with an exception.
* What went wrong:
Could not list contents of 'path_to_app/node_modules/fsevents/node_modules/.bin/node-pre-gyp'. Couldn't follow symbolic link.
```
But ```yarn``` solve this problem.
I'll create a method then will check if ```yarn``` is installed.